doc/requirements/io_class

```diff
@@ -0,0 +1,86 @@
+---
+group: IO class
+---
+
+Requirements for IO classes
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting occupancy limit
+id: occupancy_per_ioclass
+---
+
+It shall be possible to set a maximum size of an IO class.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Occupancy limit interpretation
+id: occupancy_per_ioclass_limit_interpretation
+---
+
+The product shall shall interpret max occupancy value as a percentage of
+a cache size
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed occupancy values
+id: occupancy_per_ioclass_allowed_values
+---
+
+The product shall accept values for a max size of an IO class in range from 0 to 100
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Default occupancy limit
+id: occupancy_per_ioclass_default_limit
+---
+
+When initializing a cache the software shall set default IO class's max occupancy
+to 100
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting an occupancy limit on loading a cache
+id: occupancy_per_ioclass_limit_on_load
+---
+
+When a cache state is loaded, the occupancy limit for each IO class shall be set
+to the value used before the cache has been shut down.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Display an occupancy limit value
+id: occupancy_per_ioclass_display_limit
+---
+
+It shall be possible to display the currently set value of the occupancy limit
+for each configured ioclass.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Respecting occupancy limit on a cache miss
+id: occupancy_per_ioclass_respecting_limit_on_miss
+---
+
+When an IO is a miss and the target IO class has reached it's occupancy limit,
+the software shall evict a missing number of cache blocks before inserting new
```

Comment:
maybe it would be good to state that the we shall evict from the target IO class?